microsoft-edge/devtools-guide-chromium/console/reference.md

```diff
@@ -28,120 +28,147 @@ This article is an outline of the features of the **Console**.
+This following sections describe features that change how messages are presented in the **Console** tool.  For a hands-on walkthrough, see [Inspect and filter information on the current webpage](./index.md#inspect-and-filter-information-on-the-current-webpage) in _Console overview_.
 
 
-### Turn off message grouping
+<!-- ------------------------------ -->
+#### Turn off message grouping
 
-To turn off the default message grouping behavior of the **Console**, click the **Console Settings** (![Console Settings icon](./reference-images/settings-button-icon.png)) button and then select the checkbox next to **Group similar**.  For an example, see [Log XHR and Fetch requests](#log-xhr-and-fetch-requests).
+By default, the **Console** groups similar messages together.  For example, if multiple subsequent messages are logged, only one message is displayed in the **Console**.  The message includes a count of how many times the message was logged, and you can expand the message to see all the instances.
 
+To turn off the default message grouping behavior of the **Console**, click the **Console settings** (![Console Settings icon](./reference-images/settings-button-icon.png)) button and then select the **Group similar messages in console** checkbox.
 
-### Log XHR and Fetch requests
 
-To log all `XMLHttpRequest` and `Fetch` requests to the **Console** as each happens, click the **Console Settings** (![Console Settings icon](./reference-images/settings-button-icon.png)) button and then select the checkbox next to **Log XMLHttpRequests**.
+<!-- ------------------------------ -->
+#### Log XHR and Fetch requests
 
-![Logging XMLHttpRequest and Fetch requests](./reference-images/console-xhr-fetch.png)
+To log all the network requests that are triggered by the `XMLHttpRequest` and `Fetch` JavaScript APIs:
 
-The top message in previous figure displays the default grouping behavior of the **Console**.  <!--  In the following figure, the same log is displayed after you [turn off message grouping](#turn-off-message-grouping).  -->
+1. Open the demo webpage [Inspect Network Activity Demo](https://microsoftedge.github.io/Demos/network-tutorial/) in a new window or tab.
 
-<!--
-> ##### Old Figure 9
-> How the logged `XMLHttpRequest` and `Fetch` requests look after ungrouping
-> ![How the logged XMLHttpRequest and Fetch requests look after ungrouping](./reference-images/console-xhr-fetch-all.png)
--->
+1. In the **Console** tool, click the **Console Settings** (![Console Settings icon](./reference-images/settings-button-icon.png)) button, and then elect the **Log XMLHttpRequests** checkbox.
 
-<!--todo: add example for ungrouping console items  -->
+1. In the rendered webpage, click **Get Data** to trigger a `Fetch` API request. The **Console** logs the request and response details:
```

Comment:
```suggestion
1. In the rendered webpage, click the **Get Data** button to trigger a `Fetch` API request. The **Console** logs the request and response details:
```
paint mental picture of the UI controls w/o relying on png or assuming user has right window on screen